knowledge-base/scheduler-kb-context-menu-on-appointments-and-empty-slots.md

```diff
@@ -0,0 +1,367 @@
+---
+title: Manage Appointments via Context Menu in the Scheduler
+description: How to use the context menu to add, edit, delete appointments in specific timeslots in the Scheduler.
```

Comment:
Use capital letters for component names. "Specific" is probably not a word that we need to include for SEO purposes.

```suggestion
description: How to use the Context Menu component to add, edit, and delete appointments in Scheduler timeslots.
```